<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="3" hostname="EPBYMINW2898" name="com.atm.modulefive.webdriver.advanced.mandatory.VoloTeaTest" tests="4" failures="0" timestamp="09 Jun 2017 11:22:18 GMT" time="3.067" errors="1">
  <testcase name="FlightInformation" time="0.000" classname="com.atm.modulefive.webdriver.advanced.mandatory.VoloTeaTest">
    <skipped/>
  </testcase> <!-- FlightInformation -->
  <testcase name="EnterOriginReturnDetails" time="0.000" classname="com.atm.modulefive.webdriver.advanced.mandatory.VoloTeaTest">
    <skipped/>
  </testcase> <!-- EnterOriginReturnDetails -->
  <testcase name="LoginToVoloTea" time="3.067" classname="com.atm.modulefive.webdriver.advanced.mandatory.VoloTeaTest">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at org.openqa.selenium.support.ui.ExpectedConditions.elementIfVisible(ExpectedConditions.java:305)
	at org.openqa.selenium.support.ui.ExpectedConditions.access$100(ExpectedConditions.java:44)
	at org.openqa.selenium.support.ui.ExpectedConditions$10.apply(ExpectedConditions.java:291)
	at org.openqa.selenium.support.ui.ExpectedConditions$10.apply(ExpectedConditions.java:288)
	at org.openqa.selenium.support.ui.ExpectedConditions$23.apply(ExpectedConditions.java:687)
	at org.openqa.selenium.support.ui.ExpectedConditions$23.apply(ExpectedConditions.java:680)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:209)
	at com.atm.modulefive.webdriver.advanced.utils.ActionUtility.waitForElementClickable(ActionUtility.java:19)
	at com.atm.modulefive.webdriver.advanced.pages.VoloTeaSignIn.doLogin(VoloTeaSignIn.java:29)
	at com.atm.modulefive.webdriver.advanced.mandatory.VoloTeaTest.LoginToVoloTea(VoloTeaTest.java:36)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </error>
  </testcase> <!-- LoginToVoloTea -->
  <testcase name="SearchFlights" time="0.000" classname="com.atm.modulefive.webdriver.advanced.mandatory.VoloTeaTest">
    <skipped/>
  </testcase> <!-- SearchFlights -->
</testsuite> <!-- com.atm.modulefive.webdriver.advanced.mandatory.VoloTeaTest -->
